config/routes.rb

```diff
@@ -9,6 +9,7 @@
     get '/members/unsubscribe/:message' => 'members#unsubscribe', :as => 'unsubscribe_member'
   end
   match '/members/:id/finish_signup' => 'members#finish_signup', via: [:get, :patch], :as => :finish_signup
+  match 'members/:id' => 'members#delete', :via => :delete
```

Comment:
/members is getting quite full

how about instead
```
namespace :admin do
  resources :members
end
```
Then we can have a separate controller and keep the admin versus normal membership stuff apart.